README.md

```diff
@@ -2,23 +2,8 @@
 
 ## local development
 
-```
-$ bundle install
-$ bundle exec jekyll serve --watch
-```
+Install Jekyll (or the `github-pages` gem) and then simply run:
```

Comment:
let's add at least a link to the docs on this, cause it'll save me a google next time
https://jekyllrb.com/docs/installation/

Reply:
👉 c6c52d9